static/src/stylesheets/module/content/_article-immersive.scss

```diff
@@ -18,21 +18,74 @@
     }
 }
 
-.immersive-main-media__img {
+.immersive-main-media__logo {
+    // Position above media
+    z-index: 1;
+}
+
+.immersive-main-media__logo__svg {
+    float: right;
+    margin-top: 6px;
+
+    path {
+        fill: #ffffff;
+    }
+}
+
+.immersive-main-media__media {
     width: 100%;
     // Remove inline spacing
     display: block;
 
+    .element-embed {
+        margin-top: 0;
+        margin-bottom: 0;
+        position: absolute;
+        top: 0;
+        right: 0;
+        bottom: 0;
+        left: 0;
+    }
+
+    .fenced {
+        // Override inline style coming from fence
+        height: 100% !important;
+        opacity: 0;
+        transition: .5s opacity ease-out;
+    }
+
+    .fenced-rendered {
+        opacity: 1;
+    }
+
     @supports (object-fit: cover) and (height: 1vh) and (display: flex) {
         object-fit: cover;
         position: absolute;
         top: 0;
         height: 100%;
         width: 100%;
-        z-index: 0;
     }
 }
 
+.immersive-main-media__media__loading {
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    background-color: $neutral-5;
+    color: $neutral-1;
+}
+
+.immersive-main-media__media__loading__animation {
```

Comment:
I'm never sure whether it's best to be explicit about nesting in BEM. This element only exists when it is a child of `media`. Any thoughts?